options/internal/include/mlibc/global-config.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef MLIBC_GLOBAL_CONFIG
+#define MLIBC_GLOBAL_CONFIG
+
+namespace mlibc {
+
+struct GlobalConfig {
+	GlobalConfig();
+	
+	bool debugMalloc;
+};
+
+inline const GlobalConfig &globalConfig() {
+	static GlobalConfig cached = []() -> GlobalConfig {
+		return GlobalConfig();
```

Comment:
This isn't necessary (and neither is the comment or the extranious repetition of types, a mere `static GlobalConfig cached{};` would've made this [load on invocation](https://en.cppreference.com/w/cpp/language/storage_duration#Static_local_variables))